src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicencePolicyService.kt

```diff
@@ -46,8 +35,8 @@ data class LicenceConditionChanges(
   val sequence: Int?,
   val previousText: String,
   val currentText: String?,
-  var addedInputs: List<Any>,
-  var removedInputs: List<Any>,
+  @JsonIgnore var addedInputs: List<Any>,
+  @JsonIgnore var removedInputs: List<Any>,
   val suggestions: List<SuggestedCondition> = emptyList()
```

Comment:
including these for the `PolicyVersionDifferencesTest` but excluding from the API as they aren't currently being used and causing inconveniences on the frontend.